src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly.java

```diff
@@ -21,10 +21,10 @@
 public class InputLeftCurlyCommentBeforeLeftCurly {
 
     void method1 ()
-    /* violation not reported */ { } // ok until #11410
+    /* violation not reported */ { } // violation ''{' at column 34 should be on the previous line'
```

Comment:
Am I the only who see problem to understand "'{' at column 34 should be on the previous line" message ?
All we try to convey is that `{` should be at the end of line. It is up to user where to put `{`.
in this case he might move `}` to next line.

I know that we have such message for long time, but now due to new concept of testing we see how weird it is.
Lets make new message something like: `'{' is expected at the end of line`

Reply:
ok, I make this new message. but are you thinking of replacing the message `should be on previous line` with this new one or creating a totally new one and expecting this message in this type of situation 

According to my thought it would be better to create a new message and put it in the situation where left curly and right curly at same line
```
 void method0() 
 /* comment */  {  }  // violation '{' is expected at the end of line

void method() 
/* comment */ {   // violation '{' should be on previous line
 } 

void method1() 
{ }  // violation  '{' is expected at the end of line

void method2() 
  {   //  violation  '{'  should be on previous line
  } 
 
  void method3() { } // violation  '{' is expected at the end of line

void method4() 
{ int a; }  //  violation '{'  expected at end of line

void method5() 
{     //  violation '' should be on previous line 
int a ;
} 

```
we can make implement like this

Reply:
> Am I the only who see problem to understand "'{' at column 34 should be on the previous line" message ? All we try to convey is that `{` should be at the end of line. It is up to user where to put `{`. in this case he might move `}` to next line.
> 
> I know that we have such message for long time, but now due to new concept of testing we see how weird it is. Lets make new message something like: `'{' is expected at the end of line`

I agree that messages here can be improved, @Kevin222004 please create a new issue for discussion on this and not let it block this PR.

Reply:
Message should be simple and not much guiding user to do something he does not want.

User configuring Module to control " that { is at the end of line, if not alert me".
So Module should do exactly this. 
```
option = eol

void method0() 
 /* comment */  {  }  // violation '{' should be at the end of line
```

And user will do update as he actually mean: 1) put '{' on previous line 2) move `}` to next line

------

there are move violation messages: https://checkstyle.org/config_blocks.html#LeftCurly_Violation_Messages
`line.break.after` might be better to use in our case.

Reply:
@romani @nrmancuso Issue opened at https://github.com/checkstyle/checkstyle/issues/12564